Submitted By: Joe Ciccone <jciccone@gmail.com>
Date: 2008-02-03
Initial Package Version: 2.6.23
Upstream Status: Not Submitted
Origin: Joe Ciccone
Description: Define PAGE_SIZE outside of ifdef __KEREL__ so it's not removed
             when the headers are sanatized.

diff -Naur linux-2.6.24.orig/include/asm-alpha/page.h linux-2.6.24/include/asm-alpha/page.h
--- linux-2.6.24.orig/include/asm-alpha/page.h	2008-01-24 17:58:37.000000000 -0500
+++ linux-2.6.24/include/asm-alpha/page.h	2008-02-03 12:37:59.000000000 -0500
@@ -1,16 +1,16 @@
 #ifndef _ALPHA_PAGE_H
 #define _ALPHA_PAGE_H
 
+/* PAGE_SHIFT determines the page size */
+#define PAGE_SHIFT      13
+#define PAGE_SIZE       (_AC(1,UL) << PAGE_SHIFT)
+#define PAGE_MASK       (~(PAGE_SIZE-1))
+
 #ifdef __KERNEL__
 
 #include <linux/const.h>
 #include <asm/pal.h>
 
-/* PAGE_SHIFT determines the page size */
-#define PAGE_SHIFT	13
-#define PAGE_SIZE	(_AC(1,UL) << PAGE_SHIFT)
-#define PAGE_MASK	(~(PAGE_SIZE-1))
-
 #ifndef __ASSEMBLY__
 
 #define STRICT_MM_TYPECHECKS
